content/influxdb/cloud-dedicated/reference/sql/functions/regular-expression.md

```diff
@@ -11,12 +11,60 @@ weight: 308
 influxdb/cloud-dedicated/tags: [regular expressions, sql]
 ---
 
-The InfluxDB SQL implementation uses the POSIX regular expression syntax and
-supports the following regular expression functions:
+The InfluxDB SQL implementation uses the
+[PCRE-like](https://en.wikibooks.org/wiki/Regular_Expressions/Perl-Compatible_Regular_Expressions)
+regular expression [syntax](https://docs.rs/regex/latest/regex/#syntax)
+(excluding some features such as look-around and back-references) and supports
+the following regular expression functions:
 
+- [regexp_like](#regexp_like)
 - [regexp_match](#regexp_match)
 - [regexp_replace](#regexp_replace)
 
+## regexp_like
+
+Returns true if a regular expression has at least one match in a string;
+otherwise returns false.
```

Comment:
```suggestion
True if a regular expression has at least one match in a string;
false otherwise.
```

Just a suggestion from https://developers.google.com/style/api-reference-comments#return_values